snapcraft.yaml

```diff
@@ -0,0 +1,49 @@
+name: wksctl
+version: git
+summary: Open Source Weaveworks Kubernetes Subscription
```

Comment:
I heard the S would be for System rather than Subscription?

Reply:
Yes thanks, good catch! :)